cmd/util/ledger/migrations/accounts.go

```diff
@@ -137,6 +132,73 @@ func appendKeyForAccount(accounts *state.Accounts, addressInHex string, encodedK
 	return nil
 }
 
+type view struct {
+	Parent *view
+	Ledger *led
+}
+
+func newView(payloads []ledger.Payload) *view {
```

Comment:
maybe considering passing this parameter by reference, also due to the surface of its usage, maybe defining a type aliasing as `type PayloadList []*ledger.Payload` is more convenient and less error-prone for passing by reference.

Reply:
I honestly don't think we should spend that much time here, this was a code used few months a code and only kept as a log of operation, we might change the actual logic that happened accidentally.